commit 0a6339ff3906881a0bb13c9c96bc3785cc89af56
Author: Gianfranco Trad <gianf.trad@gmail.com>
Date:   Mon Aug 19 13:06:41 2024 +0200

    Fix typo "allocateed" to allocated
    
    Fix minor typo in the fault-injection documentation,
    where the word allocated was misspelled as "allocateed".
    
    Signed-off-by: Gianfranco Trad <gianf.trad@gmail.com>
    Signed-off-by: Jonathan Corbet <corbet@lwn.net>
    Link: https://lore.kernel.org/r/20240819110641.1931-1-gianf.trad@gmail.com

diff --git a/Documentation/fault-injection/fault-injection.rst b/Documentation/fault-injection/fault-injection.rst
index 07c24710bd21..8b8aeea71c68 100644
--- a/Documentation/fault-injection/fault-injection.rst
+++ b/Documentation/fault-injection/fault-injection.rst
@@ -291,7 +291,7 @@ kernel may crash because it may not be able to handle the error.
 There are 4 types of errors defined in include/asm-generic/error-injection.h
 
 EI_ETYPE_NULL
-  This function will return `NULL` if it fails. e.g. return an allocateed
+  This function will return `NULL` if it fails. e.g. return an allocated
   object address.
 
 EI_ETYPE_ERRNO